Native mobile VR module is lost (3.1.dev)

<!-- Please search existing issues for potential duplicates before filing yours:
https://github.com/godotengine/godot/issues?q=is%3Aissue
-->

**Godot version:**
<!-- Specify commit hash if non-official. -->
Godot 3.1.dev from current master 03e11c13b

**OS/device including version:**
<!-- Specify GPU model and drivers if graphics-related. -->
MacOSX 10.13.6

**Issue description:**
<!-- What happened, and what was expected. -->
I have made minimal VR project where I try to initialize "Native mobile" module. There is my `_ready` function:
```
func _ready():
    print("There are %d VR interfaces" % ARVRServer.get_interface_count())
    for interface in ARVRServer.get_interfaces():
        print("VR: %s" % interface.get_name())
    var arvr_interface = ARVRServer.find_interface("Native mobile")
    if arvr_interface:
        if arvr_interface.initialize():
            get_viewport().arvr = true
        else:
            print("VR not initialized!")
    else:
        print("VR not found!")
```

When I started on desktop (or on android). It can not find any VR interface:
```
OpenGL ES 3.0 Renderer: NVIDIA GeForce GT 1030 OpenGL Engine
There are 0 VR interfaces
ERROR: find_interface: Condition ' idx == -1 ' is true. returned: __null
   At: servers/arvr_server.cpp:208.
VR not found!
```

I compiled godot with `scons platform=osx target=release_debug optimize=size use_lto=yes tools=yes bits=64 -j12`
Also I checked that mobile vr module is enabled by default:
```
module_mobile_vr_enabled: Enable module 'mobile_vr' (yes|no)
    default: True
    actual: True
```

**Steps to reproduce:**
Start the project.

**Minimal reproduction project:**
<!-- Recommended as it greatly speeds up debugging. Drag and drop a zip archive to upload it. -->
https://www.dropbox.com/s/9hd6dzz2hktv4ew/vrtest2.zip?dl=0

When I changed `can_build` function (to return True) I can not build godot nor for osx neigther android.
```
core/class_db.h:142:17: error: allocating an object of abstract class type 'MobileVRInterface'
                return memnew(T);
                              ^
core/class_db.h:175:23: note: in instantiation of function template specialization 'ClassDB::creator<MobileVRInterface>' requested here
                t->creation_func = &creator<T>;
                                    ^
modules/mobile_vr/register_types.cpp:36:11: note: in instantiation of function template specialization 'ClassDB::register_class<MobileVRInterface>' requested here
        ClassDB::register_class<MobileVRInterface>();
                 ^
./servers/arvr/arvr_interface.h:91:15: note: unimplemented pure virtual method 'is_initialized' in 'MobileVRInterface'
        virtual bool is_initialized() const = 0; /* returns true if we've initialized this interface */
                     ^
In file included from modules/mobile_vr/register_types.cpp:33:
In file included from modules/mobile_vr/mobile_interface.h:34:
In file included from ./servers/arvr/arvr_interface.h:36:
In file included from ./scene/main/viewport.h:35:
In file included from ./scene/main/node.h:39:
In file included from ./scene/main/scene_tree.h:34:
In file included from core/io/multiplayer_api.h:34:
In file included from ./core/io/networked_multiplayer_peer.h:34:
In file included from core/io/packet_peer.h:34:
In file included from core/io/stream_peer.h:34:
./core/reference.h:268:14: error: allocating an object of abstract class type 'MobileVRInterface'
                ref(memnew(T));
                           ^
modules/mobile_vr/register_types.cpp:39:12: note: in instantiation of member function 'Ref<MobileVRInterface>::instance' requested here
        mobile_vr.instance();
```

--- a/modules/mobile_vr/config.py
+++ b/modules/mobile_vr/config.py
@@ -1,6 +1,5 @@
 def can_build(env, platform):
-    # should probably change this to only be true on iOS and Android
-    return False
+    return True
 
 def configure(env):
     pass

--- a/modules/mobile_vr/mobile_vr_interface.cpp
+++ b/modules/mobile_vr/mobile_vr_interface.cpp
@@ -28,7 +28,7 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#include "mobile_interface.h"
+#include "mobile_vr_interface.h"
 #include "core/os/input.h"
 #include "core/os/os.h"
 #include "servers/visual/visual_server_global.h"
@@ -279,7 +279,7 @@ bool MobileVRInterface::is_stereo() {
 	return true;
 };
 
-bool MobileVRInterface::is_initialized() {
+bool MobileVRInterface::is_initialized() const {
 	return (initialized);
 };
 

--- a/modules/mobile_vr/mobile_vr_interface.h
+++ b/modules/mobile_vr/mobile_vr_interface.h
@@ -134,7 +134,7 @@ public:
 	virtual StringName get_name() const;
 	virtual int get_capabilities() const;
 
-	virtual bool is_initialized();
+	virtual bool is_initialized() const;
 	virtual bool initialize();
 	virtual void uninitialize();
 
@@ -150,4 +150,4 @@ public:
 	~MobileVRInterface();
 };
 
-#endif // MOBILE_VR_INTERFACE_H
+#endif // !MOBILE_VR_INTERFACE_H

--- a/modules/mobile_vr/register_types.cpp
+++ b/modules/mobile_vr/register_types.cpp
@@ -30,7 +30,7 @@
 
 #include "register_types.h"
 
-#include "mobile_interface.h"
+#include "mobile_vr_interface.h"
 
 void register_mobile_vr_types() {
 	ClassDB::register_class<MobileVRInterface>();

